apis_configs/pidgin_settings.py

```diff
@@ -0,0 +1,14 @@
+from pidgin.app import app
+from os import environ
+import config_helper
+
+APP_NAME='pidgin'
+def load_json(file_name):
+  return config_helper.load_json(file_name, APP_NAME)
+
+#conf_data = load_json('creds.json')
+config = app.config
+
+config['API_URL'] = 'http://peregrine-service/v0/submission/graphql/'
+#app_init(app)
+application = app
```

Comment:
Can we get rid of this, and just have the default `settings.py` in pidgin load a json file for configuration?  You can pull the `config_helper` into `pyutils`, so it's generally available ...

BTW - if this is green field - I hope we're running python 3 ...

Reply:
So it looks like the only thing this file does is set 'API_URL' - you can get rid of the `config_helper` stuff, and yeah - would be cool to not even need this file ...